CMakeLists.txt

```diff
@@ -6,6 +6,16 @@ include(GenerateExportHeader)
 include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
 
+if (NOT DEFINED CMAKE_ARCHIVE_OUTPUT_DIRECTORY)
+    set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+endif()
+if (NOT DEFINED CMAKE_LIBRARY_OUTPUT_DIRECTORY)
+    set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+endif()
+if (NOT DEFINED CMAKE_RUNTIME_OUTPUT_DIRECTORY)
+    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+endif()
+
```

Comment:
Hijacking global CMake variables is strongly discouraged for libraries in general. It can lead to situations where you just add a library and then your whole build starts behaving differently, since these are global. Instead, use `set_target_property` with `LIBRARY_OUTPUT_DIRECTORY` etc. to do this on a per-target basis.

Reply:
@athre0z don't worry, before updating I always check if I have to change the Debian build script, but thanks for keeping me updated :)

Reply:
I removed that part for now. Will create a separate PR after doing some research.